components/projectQueryForm.js

```diff
@@ -0,0 +1,172 @@
+import React, { Component } from 'react'
+import firebase from 'firebase'
+import moment from 'moment-timezone'
+import _ from 'lodash'
+import 'common/initializeFirebase.js'
+import Div from 'components/core/div.js'
+import Form from 'components/core/form.js'
+import Button from 'components/core/button.js'
+import ResponsiveContainer from 'components/core/responsiveContainer'
+import { H3 } from 'components/core/headerText'
+import styled from 'styled-components'
+
+const OuterContainer = styled(ResponsiveContainer)`
+  #innerContainer {
+    align-items: center;
+    border: solid 1px #dedee0;
+    background: white;
+    display: flex;
+
+    @media (min-width: ${props => props.theme.screenLgMin}px) {
+      max-width: ${props => props.theme.screenLgWidth / 2.5}px;
+      min-width: ${props => props.theme.screenLgWidth / 2.5}px;
+      margin-left: auto;
+      margin-right: auto;
+    }
+
+    @media (max-width: ${props => props.theme.screenLgMin}px) {
+      padding-left: 25px;
+      padding-right: 25px;
+    }
+  }
+
+  margin-top: 50px;
+  flex-direction: column;
+  align-items: center;
+`
+
+const FormContainer = styled(Div)`
+  width: 100%;
+  flex-direction: column;
+  align-items: center;
+`
+const FormHeaderContainer = styled(Div)`
+  width: 100%;
```

Comment:
generally we only use %'s with flexbox as a last result. 

If you want something to take up all possible space, it's usually a cocktail of using `align-items`/`justify-content`: `stretch` or maybe `flex: 1`.

You can try both (you might need it to do it at multiple levels of the hierarchy.